### PR TITLE
ideas for additions to project

### PR DESCRIPTION
**Pull request by andrewpaster**
changed long timestamp to long long timestamp

### CODE REVIEW

**Comment by andrewpaster** [comment]
latest commit adds RMSE output at end of code. Students still have to finish the RMSE function. NIS also gets outputted into the output file and the output file now has header columns.

**Comment by cameronwp** [comment]
this needs to be split into two PRs. one should handle the `long` -> `long long` transition, which will get approved ASAP. the other should handle the addition of ground truth. I want to take a closer look at the latter and I don't want to block fixing `long long` on it.

so basically, pull the last commit out and put it into its own PR.

File: src/tools.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Tools::~Tools() {}
+
+VectorXd Tools::CalculateRMSE(const vector<VectorXd> &estimations,
+                              const vector<VectorXd> &ground_truth) {
+  /**
+  TODO:
+    * Calculate the RMSE here.
+  */
+}
```
Comment (cameronwp) [inline src/tools.cpp:18]: nit: can we add a newline here?

File: src/ukf.h [hunk 2]
```diff
@@ -29,7 +33,7 @@ class UKF {
   MatrixXd Xsig_pred_;
 
   ///* time when the state is true, in us
-  long time_us_;
+  long long time_us_;
```
Comment (cameronwp) [inline src/ukf.h:36]: isn't this in the other PR that just got merged?

**Comment by cameronwp** [comment]
need to kill this PR. half of it has already been merged into master in a different PR. I'm going to cherry pick 	52efb0e out and resubmit as a new PR

**Pull request closed without merging**